h2o-py/tests/testdir_algos/isoforextended/pyunit_isoforextended_smoke.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import print_function
+import sys, os
+sys.path.insert(1, os.path.join("..","..",".."))
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.extended_isolation_forest import H2OExtendedIsolationForestEstimator
+
+
+def extended_isolation_forest():
+    print("Extended Isolation Forest Smoke Test")
+
+    train = h2o.import_file(pyunit_utils.locate("smalldata/anomaly/single_blob.csv"))
+
+    eif_model = H2OExtendedIsolationForestEstimator(ntrees=7, seed=12, sample_size=5)
+    eif_model.train(training_frame=train)
+    anomaly_score = eif_model.predict(train)
+    anomaly = anomaly_score['anomaly_score'].as_data_frame(use_pandas=True)["anomaly_score"]
+
+    print(eif_model)
+    
+    assert (abs(anomaly[0] - 0.54)) <= 0.01, "Not expected output: Expected value is about 0.54"
```

Comment:
Leave comment with link to the source paper here, for others to know where the values come from.